app/controllers/charges_controller.rb

```diff
@@ -63,18 +63,82 @@ def create
       return
     end
 
-    trigger_payment_mailer(service.charge, outstanding_before_charge, charge_amount)
+    trigger_reservation_payment_mailer(service.charge, outstanding_before_charge, charge_amount)
 
     message = "Thank you for your #{charge_amount.format} payment"
     (message += ". Your #{@reservation.membership} membership has been paid for.") if @reservation.paid?
 
     redirect_to reservations_path, notice: message
   end
 
+  def create_group_charge
+    @transaction_cart = Cart.find_by(id: params[:buyable])
+    charge_amount = Money.new(@transaction_cart.subtotal_cents)
+
+    successful = ActiveRecord::Base.transaction(joinable: false, requires_new: true) do
+
+      service = Money::ChargeCustomer.new(
```

Comment:
How hard would it be to send the actual items through here to Stripe so that they show up as separate line items in Stripe?

Reply:
I agree that  such would be much, much better in a lot of ways.  My current understanding is that to do that, we'd have to make a series of individual Stripe calls.  But maybe when we upgrade Stripe, that will become more possible? 

I'd be happy to do more research to see if there's some other way.

Reply:
It turns out that Line Items are absolutely A Thing in the new version of Stripe, and IMO, that functionality would really improve matters for us. 

[https://stripe.com/docs/payments/checkout/migration](url)